doc/website/advanced/configuration-guide.md

```diff
@@ -31,7 +31,7 @@ With that change, the footprint of the management segment is reduced to ~52.7 MB
 ## :material-memory: Configuring Mempools for RouDi
 
 RouDi supports several shared memory segments with different access rights, to limit the read and write access between different applications. Inside of these segments reside mempools where the user payload data for transfer is stored.
-Based on the [conceptual guide](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/conceptual-guide.md) the end-user may want to configure the mempools with the number of chunks and their size.
+Based on the [shared memory concept article](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/shared-memory-communication.md) the end-user may want to configure the mempools with the number of chunks and their size.
```

Comment:
Actually I don't understand why we refer to the article.

Reply:
Hmm :thinking: I thought of it as getting more background info of now. Thinking again, I guess you're right. Removed.